digits/model/tasks/torch_train.py

```diff
@@ -110,8 +110,8 @@ def before_run(self):
 
     def create_mean_file(self):
         filename = os.path.join(self.job_dir, constants.MEAN_FILE_IMAGE)
-        # don't recreate file if it already exists
-        if not os.path.exists(filename):
+        # don't recreate file if it already exists, unless user has specified a non-db mean path
+        if not os.path.exists(filename) or self.mean_path is not None:
```

Comment:
I might be missing something but I don't understand the need to change this condition. The mean file must be created once per model job, either from the dataset mean or from the custom mean. In both cases `self.get_mean_path()` returns the correct file. Note that `self.job_dir` is the model job dir. This is different from the dataset job dir.